tests/cover/test_simple_characters.py

```diff
@@ -84,6 +84,23 @@ def test_find_something_rare():
         find(st, lambda c: unicodedata.category(c) != 'Zs')
 
 
+def test_whitelisted_characters_alone():
+    with pytest.raises(InvalidArgument):
+        characters(whitelist_characters=u'te02тест49st').example()
+
+
+def test_whitelisted_characters_override():
+    good_characters = u'teтестst'
+    st = characters(min_codepoint=ord('0'), max_codepoint=ord('9'),
+                    whitelist_characters=good_characters)
+
+    st.filter(lambda c: c in good_characters).example()
+    st.filter(lambda c: c in '0123456789').example()
+
+    with pytest.raises(NoSuchExample):
+        find(st, lambda c: c not in good_characters + '0123456789')
```

Comment:
Whoops!  Yes, it's safe just to change to `NoExamples` - as you might be able to guess from the traceback, `strategy.example()` is actually implemented in terms of `find`; the only difference is that if we *do* find an example with the new version we don't try to minimize it.